documentation-processor/src/main/java/com/koval/resolver/processor/documentation/core/DocDataSetCreator.java

```diff
@@ -73,20 +77,20 @@ public void create() throws IOException {
                 String docPageKey = KEY_PREFIX + pageIndex;
                 pageIndex++;
 
-                dataSetOutput.print(docPageKey);
-                dataSetOutput.print(SEPARATOR);
-                dataSetOutput.println(TextUtil.simplify(docPage.getValue()));
+                dataSetBufferedWriter.write(docPageKey);
+                dataSetBufferedWriter.write(SEPARATOR);
+                dataSetBufferedWriter.write(TextUtil.simplify(docPage.getValue()) + "\n");
```

Comment:
It's better to add new line with dataSetBufferedWriter.write("\n") in order to prevent implicitly creating a new String object.

Reply:
Ok